manifests/gateways/istio-egress/templates/deployment.yaml

```diff
@@ -109,6 +109,15 @@ spec:
         {{- end }}
           - --controlPlaneAuthPolicy
           - NONE
+          - --discoveryAddress
```

Comment:
This doesn't do anything in master, it is reading from mesh config directly. 1.5 its needed though

Reply:
can you remove after this merges?

Reply:
Yup - https://github.com/istio/istio/issues/22050